src/pf-utils/pf-utils.js

```diff
@@ -38,4 +38,76 @@ let I18nUtil = function() {
     self.mixin = mixin;
   };
 };
-export let i18n = new I18nUtil();
+let i18n = new I18nUtil();
+export {i18n};
+
+/**
+ * --------------------------------------------------------------------------
+ * PfUtil
+ * Native Javascript for Bootstrap 3 | Internal Utility Functions
+ * --------------------------------------------------------------------------
+ */
+
+class PfUtil {
```

Comment:
+1 for moving each related Util to its own file. I'm not entirely sold on naming it NativeBootstrapUtil as we'll be changing this quite a bit, and likely appending to it. I would think that a large number of common routines that used to be in jQuery (like addClass, removeClass, getClosest) now being implemented in Vanilla js would fit into the same logical utility file (and I guess my preference is just PfUtils for that). Let me know if this works and I'm happy to split it out.